earn/src/components/advanced/BorrowMetrics.tsx

```diff
@@ -268,14 +265,34 @@ export function BorrowMetrics(props: BorrowMetricsProps) {
       </MetricsGridUpper>
       <MetricsGridLower>
         <HealthMetricCard health={marginAccount.health || 0} />
-        <HorizontalMetricCard label='Liquidation Distance' value={liquidationDistanceText} />
         <HorizontalMetricCard
           label='Daily Interest Owed'
           value={`${formatTokenAmount(dailyInterest0, 2)} ${marginAccount.token0.symbol},  ${formatTokenAmount(
             dailyInterest1,
             2
           )} ${marginAccount.token1.symbol}`}
         />
+        {mostRecentModifyHash && (
+          <HorizontalMetricCard label='Last Modified'>
+            <Text size='M'>
+              <a className='underline text-purple' rel='noreferrer' target='_blank' href={mostRecentModifyUrl}>
+                {mostRecentModifyTimeStr}
+              </a>{' '}
+              using {mostRecentManagerName ? 'the ' : 'an '}
+              <a className='underline text-purple' rel='noreferrer' target='_blank' href={mostRecentManagerUrl}>
+                {mostRecentManagerName ?? 'unknown manager'}
+              </a>
+            </Text>
+          </HorizontalMetricCard>
+        )}
+        <HorizontalMetricCard label='Custom Tag'>
+          <Text size='M'>{marginAccount.userDataHex}</Text>
```

Comment:
Some of my "Custom Tag"'s are just 0x00, why is that?

Reply:
Because you haven't set a custom tag. Currently we use the tags as a way of indicating what was created through the Markets page vs the Boost page. Soon I will add a button that lets people customize them on the Advanced page.